tests/end-to-end/backup-restore-test/Makefile

```diff
@@ -9,8 +9,8 @@ build:
 .PHONY: build-image
 build-image:
 	mkdir -p build-temp
-	cp ../../../docs/backup/docs/assets/all-backup.yaml build-temp
-	cp ../../../docs/backup/docs/assets/system-backup.yaml build-temp
+	cp ./deploy/all-backup.yaml build-temp
```

Comment:
please don't change these file paths. we wanted to use the files as they are used in the documentation